tools/awps-tunnel/client/src/panels/RequestHistory.tsx

```diff
@@ -126,28 +128,93 @@ export function RequestHistory(props: RequestHistoryProps) {
   );
 }
 
+const renderContent = (message: {headers: Record<string, string>, content: string}): ReactNode => {
+  if(message.headers['Content-Type'] === 'application/json' || message.headers['Content-Type'] === 'text/json'){
+    try {
+      const parsedJson = JSON.parse(message.content);
+      return <div><ReactJson src={parsedJson} collapsed={1}/></div>;
+    } catch (error) {
+    }
+  }
+  return <div className="m-2" style={{ whiteSpace: "pre-wrap" }}>
+    {message.content}
+  </div>
+};
+
+function revertRawTextToMessage(rawText: string):{ headers: Record<string, string>, content: string }{
+  const lines: string[] = rawText.split('\n').map(line => line.trim());
+  const emptyIndex: number = lines.indexOf('');  // 找到空行，它分隔了headers和body
+  let headers: Record<string, string> = {};
+  let content: string = '';
+  for (let i: number = 0; i < emptyIndex; i++) {
+    const splitIndex: number = lines[i].indexOf(':');
+    if (splitIndex !== -1) {
+      const key: string = lines[i].substring(0, splitIndex).trim();
```

Comment:
add contentType to the returned object and compare key.toLower with "content-type" here